NSIS: Fixed italian.nsi after 4.3.0.1 update

alessandrosimonelli opened the pull request:
Fixed typos and errors. "%n%n" appeared twice and was visible during setup.

bovirus commented:
@alessandrosimonelli 

Why did you change the meaning of strings?
I fixed the problem about %n here
https://github.com/qbittorrent/qBittorrent/pull/13610
I didn't understand why you changed the meaning of other strings.
I define when you talk "application" that is relative to qbittorennt because just say "application" could be not clear for all.

**_I'm not agree about some of your changes._**

alessandrosimonelli commented:
@bovirus I'm going to explain each change i made.

`inst_dekstop`: the translation was "create iconS on desktop". It should be singular (icona, not icone). I changed it into "collegamento" (shortcut), which i think is more appropriate in. "Crea icona sul Desktop" would be fine too.

`inst_startmenu`: the translation was "create program group" (crea gruppo programmi), which I believe makes no sense.

`inst_pathlimit`: your translation was "disable Windows path limit", you forgot "lunghezza" (length). Then there was a typo. It's "caratteri", not "caratterin".

`inst_warning`: there is no need to specify qBittorrent twice, it's clear what should be closed. If you prefer a clearer version while avoiding repetition "Chiudilo prima di procedere con l'installazione." would work too.

`inst_uninstall_question`: same as above. You are installing qBittorrent. It's clear that the previous version of the same program is going to be uninstalled. It's not mentioned in the English string either. There is a typo: "rimanno" is not a verb, the word you're looking for is "rimarranno".

`inst_unist`: same as above.

`launch_qbt`: added the dot back since it's present in the English string.

`inst_requires_64bit`: "qBittorrent" not needed. You know you're running a qBittorrent installer, as it's displayed in the top bar. Also, not present in the English string

`uninst_warning`: there is no need to specify qBittorrent twice, it's clear what should be closed. If you wanted a clearer version while avoiding repetition "Chiudilo prima di procedere con la disinstallazione." would work too.

About CR/LF, no other translation uses it so I didn't think it was needed.

bovirus commented:
I didn't agree about it.

Other translations don't use CR/LF because other gusys not knows the use of CR/LF.
If we can improve Italian translation (using of CR/LF) I didn't udnerstand why you cannot do it because other language not use it.
The multi-perod strings are more readable if you use multi-period on different line for each period.
It's true for Italian but also for other language (english included).
Try to ude the CR/LF and check if the strings is more clear than without CR/LF.

If you can increase clearance that you are talking about qbt why don't explain it?
So close the application without explain that you are referring about QBT can create misunderstanding.

Path limit is path limit referring lenght of path limit.
It's you addition that has no sense.

If it the translation is "Create program group" you have to translate it and not use a different words with different meaning.

launch_qbt: it's wrong in English because in any program is an option at the end that don't need ".".

When you want you follow the english master words and sometimes when you don't like Eenglish strings don't use the same translation.

It this a non sense.

You approach  it's only a theorical approach without a real know about software platform.

alessandrosimonelli commented:
![image](https://user-images.githubusercontent.com/45544818/96905730-23e9aa80-1499-11eb-8ab6-e378d3e0bef4.png)
Regarding **CR/LF**, the text above seems cleary readable, I don't think there is the need to start a new line. Generally, I think it's best to follow the formatting of the original strings (that's why I also added the dot back). As these are just formatting changes that apply to almost every language, you could have edited all translations to see if the devs liked your changes while keeping all translations consistent.

About the **qBittorrent repetition**, I think that my translation is already pretty clear. English string is "qBittorrent is running. Please close the application before installing.", no repetition. Your edit translates back to "_qBittorrent_ is running. Please close the _qBittorrent_ application before installing.". My clearer edit (which I don't feel there is a need for, as it's already clear what you should do) is "qBittorrent is running. Please close it before installing." which in Italian is "[...] Chiudilo prima di procedere con l'installazione."

About **path**, it says it's a _length_ limit. "Disable Windows path _length_ limit" -> "Disabilita limite _lunghezza_ percorsi Windows". Later in the same line, my translation is "limite MAX_PATH di 260 caratteri", which means that there is a limit called MAX_PATH set to 260 characters.

"Crea gruppo programmi" translates back to "Create program group". It makes no sense because the corresponding English string is "Create Start Menu Shortcut", which in Italian is "Aggiungi al menu Start". In the original translation it's not clear what would happen if someone ticked the relative box, the Start Menu is not mentioned at all.

I think I've explained my choices, let's see what others think!

bovirus commented:
![Snap13](https://user-images.githubusercontent.com/1262554/96957277-7f985000-14fa-11eb-8d5e-66b4097e4d3f.jpg)

As you can see the multi-line string with CR/LF is more readable then single string without CR/LF.

If you work as translator you know that the attention of brain follow the line of thinking and autowrap line has less attention then separate line.

I repeat the question: what's the negative point to use CR/LF to separate multi-period line? For me nothing.

About "applicatioN" is string that can be replaced by qBittorren (application name).
Same space. More clear and diorect.
